ui/component/settingWalletServer/view.jsx

```diff
@@ -107,13 +106,12 @@ function SettingWalletServer(props: Props) {
 
   return (
     <React.Fragment>
-      <label>{__('Wallet servers')}</label>
```

Comment:
I think this label still needs to be here, or some text above the radio choices explaining what they are.

Reply:
Please also put a link to a FAQ article (https://lbry.com/faq/wallet-servers - making it shortly) next to the label or in help text below the radio choices.